classes/help.php

```diff
@@ -207,19 +206,19 @@ function print_page() {
 
 												<tr>
 													<td>
-														<?php _e( 'To properly add a new location, you must enter one or both of the following:', 'SimpleMap' ); // XSS ok. ?>
+														<?php _e( 'To properly add a new location, you must enter one or both of the following:', 'simplemap' ); // XSS ok. ?>
 														<br/>
 														<span
-															style="padding-left: 20px;"><?php _e( '1. A full address', 'SimpleMap' ); // XSS ok. ?></span><br/>
+															style="padding-left: 20px;"><?php _e( '1. A full address', 'simplemap' ); // XSS ok. ?></span><br/>
 														<span
-															style="padding-left: 20px;"><?php _e( '2. A latitude and longitude', 'SimpleMap' ); // XSS ok. ?></span><br/>
-														<?php _e( 'If you enter a latitude and longitude, then the address will not be geocoded, and your custom values will be left in place. Entering an address without latitude or longitude will result in the address being geocoded before it is submitted to the database.', 'SimpleMap' ); // XSS ok. ?>
+															style="padding-left: 20px;"><?php _e( '2. A latitude and longitude', 'simplemap' ); // XSS ok. ?></span><br/>
+														<?php _e( 'If you enter a latitude and longitude, then the address will not be geocoded, and your custom values will be left in place. Entering an address without latitude or longitude will result in the address being geocoded before it is submitted to the database.', 'simplemap' ); // XSS ok. ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 353 characters

<!-- Comment by [SideCI](https://sideci.com) -->